Joplin_install_and_update.sh

```diff
@@ -180,12 +180,12 @@ SANDBOXPARAM=""
 if command -v lsb_release &> /dev/null; then
   DISTVER=$(lsb_release -is) && DISTVER=$DISTVER$(lsb_release -rs)
   DISTCODENAME=$(lsb_release -cs)
+  DISMAJOR=$(lsb_release -rs|cut -d. -f1)
   #-----------------------------------------------------
   # Check for "The SUID sandbox helper binary was found, but is not configured correctly" problem.
   # It is present in Debian 1X. A (temporary) patch will be applied at .desktop file
   # Linux Mint 4 Debbie is based on Debian 10 and requires the same param handling.
-  if [ $DISTVER =~ Debian1. ] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ]
-  then
+  if [ $DISTVER =~ Debian1. ] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ] || [ "$DISTVER" = "CentOS"] && [ "$DISMAJOR" = 7 ]
```

Comment:
How about version 5 and 6 or future 8 and 9? Will the script need to be changed for all versions?

Reply:
I am not sure, but CentOS8 doesn't need it. reference: this [website](http://serverarekore.blogspot.com/2021/01/centos-8-streamjoplin.html?m=0)

Reply:
Ok, in that case we are fine. CentOS 5 and 6 should not be used anyway anymore. And if there's really someone out there who uses CentOS 6, we can add that later.

Reply:
As mentioned in the previous comment, please change `DISMAJOR` to `DISTMAJOR`.
And also enclose the 7 in double quotes for the test.